interface conversion: interface {} is float64, not string

### Confirmation

- [X] My issue isn't already found on the issue tracker.
- [X] I have replicated my issue using the latest version of the library and it is still present.

### cf-terraforming version

cf-terraforming v0.1.1

### Expected outcome

The terraform resource definition for cloudflare_logpush_job.

### Actual outcome

```
panic: interface conversion: interface {} is float64, not string

goroutine 1 [running]:
github.com/cloudflare/cf-terraforming/internal/app/cf-terraforming/cmd.generateResources.func1(0x20b4ce0, 0xc000486
2a0, 0x0, 0x6)
        /home/runner/work/cf-terraforming/cf-terraforming/internal/app/cf-terraforming/cmd/generate.go:522 +0x8c2b
github.com/spf13/cobra.(*Command).execute(0x20b4ce0, 0xc000486240, 0x6, 0x6, 0x20b4ce0, 0xc000486240)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x20b51e0, 0xc0002dff78, 0x1006045, 0xc0000480b8)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/cloudflare/cf-terraforming/internal/app/cf-terraforming/cmd.Execute()
        /home/runner/work/cf-terraforming/cf-terraforming/internal/app/cf-terraforming/cmd/root.go:30 +0x31
main.main()
        /home/runner/work/cf-terraforming/cf-terraforming/cmd/cf-terraforming/main.go:8 +0x25
```

### Steps to reproduce

Run the command:
```
cf-terraforming generate --resource-type "cloudflare_logpush_job" --email=<your email> --key=<api-key> --account=<cf-account-id> --zone=<cf-zoneid> > logpush_job.tf
```

### References

_No response_

--- a/internal/app/cf-terraforming/cmd/generate.go
+++ b/internal/app/cf-terraforming/cmd/generate.go
@@ -519,7 +519,15 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 			if os.Getenv("USE_STATIC_RESOURCE_IDS") == "true" {
 				resourceID = "terraform_managed_resource"
 			} else {
-				resourceID = fmt.Sprintf("terraform_managed_resource_%s", structData["id"].(string))
+				id := ""
+				switch structData["id"].(type) {
+				case float64:
+					id = fmt.Sprintf("%f", structData["id"].(float64))
+				default:
+					id = structData["id"].(string)
+				}
+
+				resourceID = fmt.Sprintf("terraform_managed_resource_%s", id)
 			}
 
 			output += fmt.Sprintf(`resource "%s" "%s" {`+"\n", resourceType, resourceID)

